src/App.jsx

```diff
@@ -1,35 +1,27 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import "./App.css";
 
-function App() {
-  const [count, setCount] = useState(0)
+const todoList = [
+  { id: 1, title: "Drumming" },
+  { id: 2, title: "Coding" },
+  { id: 3, title: "Socializing" },
+];
```

Comment:
I like the inline array items here – it really helps in keeping your list visually compact and readable.